registration-processor-default.properties

```diff
@@ -998,3 +998,8 @@ mosip.registration.processor.lostrid.max-registration-date-filter-interval=30
 uingenerator.lost.packet.allowed.update.fields=phone,email,permanentAddress
 ##timeout in milliseconds for health check registrer
 mosip.regproc.health-check.handler-timeout=2000
+## List of old reg client versions with exceptionbiometrics contains applicant json object, this is required to take care of the backward compatibility checks
+mosip.regproc.packet.classifier.tagging.exceptionbiometrics.reg-client-versions=1.1.3,1.1.4
+##Current reg client version used if in case reg client version not present in packet
+mosip.regproc.packet.classifier.tagging.reg-client-current-version=1.2.1-SNAPSHOT
```

Comment:
Last to be removed, and previous to be renamed as mosip.regproc.packet.classifier.tagging.exceptionbiometrics.before-exception-metainfo-change.reg-client-versions